{"event": "verbose", "uuid": "ce9fb1b6-f2ff-4c8f-8a5c-23b875d3dfa8", "counter": 7, "stdout": "\u001b[1;35mpackages/ansible/plugins/connection/vmware_tools.py) as it seems to be invalid:\u001b[0m", "start_line": 6, "end_line": 7, "runner_ident": "0ce4ed8f-7f4e-4b3d-8c2d-b4434adf49bd", "created": "2023-05-16T05:59:26.020090"}